Makefile

```diff
@@ -259,3 +261,62 @@ datasets/enwiki.draft_quality.201608-201701.with_cache.json.bz2: \
 	articlequality extract_from_text \
 	  draftquality.feature_lists.enwiki.draft_quality \
 	  --verbose | bzip2 -c > $@
+
+#################### Portuguese Wikipedia #######################
+datasets/ptwiki.draft_quality.201903202003.json:
+	wget -qO- https://quarry.wmflabs.org/run/444292/output/0/json-lines?download=true > $@
+
+datasets/ptwiki.draft_quality.balanced_3k.json.bz2: \
+		datasets/ptwiki.draft_quality.201903202003.json
+	(cat $< | grep '"draft_quality": "OK"' | shuf -n 1600; \
+	 cat $< | grep -v '"draft_quality": "OK"') | \
+	shuf | bzip2 -c > $@
+
+datasets/ptwiki.draft_quality.balanced_3k.with_text.json.bz2: \
+		datasets/ptwiki.draft_quality.balanced_3k.json.bz2
+	bzcat $< | \
+	revscoring fetch_text --host https://pt.wikipedia.org --threads 4 \
+	 --verbose | bzip2 -c > $@
+
+datasets/ptwiki.draft_quality.balanced_3k.with_cache.json.bz2: \
+		datasets/ptwiki.draft_quality.balanced_3k.with_text.json.bz2
+	bzcat $< | \
+	articlequality extract_from_text \
+	  draftquality.feature_lists.ptwiki.draft_quality \
+	  --verbose | bzip2 -c > $@
+
+tuning_reports/ptwiki.draft_quality.md: \
+		datasets/ptwiki.draft_quality.balanced_3k.with_cache.json.bz2
+	bzcat $< | \
+	revscoring tune \
+	  config/classifiers.params.yaml \
+	  draftquality.feature_lists.ptwiki.draft_quality \
+	  draft_quality \
+		roc_auc.macro \
+		--pop-rate '"OK"=0.97080700532 ' \
+		--pop-rate '"spam"=0.0095278372' \
+		--pop-rate '"unsuitable"=0.01966515747 ' \
+		--scale --center \
+	  --cv-timeout=90 \
+	  --debug > $@
+
+models/ptwiki.draft_quality.gradient_boosting.model.bz2: \
+		datasets/ptwiki.draft_quality.balanced_3k.with_cache.json.bz2
+	bzcat $< | \
+	revscoring cv_train \
+	  revscoring.scoring.models.GradientBoosting \
+	  draftquality.feature_lists.ptwiki.draft_quality \
+	  draft_quality \
+	  -p 'n_estimators=300' \
+	  -p 'learning_rate=0.1' \
+	  -p 'max_depth=5' \
+	  -p 'max_features="log2"' \
```

Comment:
Change this to the best tuning: n_estimators=700, max_depth=7, learning_rate=0.01, max_features="log2"